### ISSUE TITLE
Testing data folders should be untracked

### ISSUE THREAD

**Issue by girgink** [ITC-CRIB/fairly#14]
`dummy_dataset` and `vcr_cassettes` seem to be tracked by git. They should be ignored, otherwise running unit tests result are marked as none-staged changes.

**Comment by jurra** [comment]
I agree @girgink the problem is I was recording some of the data in the vcr cassettes, and I was also using some random generation of values. I think this is the main reason why I decided to store them as fixtures. Fixtures are also common in unit testing, so I think this is more a matter of style setting up the testing than a bad practice. This datasets are extremely light.

**Comment by jurra** [comment]
See PR #17 it also solves the problem of storing the testing dataset in the repo
